.spec/route_spec.rb

```diff
@@ -42,7 +42,11 @@
       expect(response).to be_a(Net::HTTPSuccess)
     end
 
+<<<<<<< HEAD
     it 'returns an empty string' do
+=======
+    it "returns an empty string" do
+>>>>>>> 87f9e441 ([Crystal] Update Athena to 0.13 (#3926))
```

Comment:
@waghanza Some merge conflicts made it in here.

Reply:
seems, to will fix

it think `overcommit` could help me to avoid this 😛